kiwi-web.yml

```diff
@@ -69,7 +71,88 @@ spec:
           ports:
             - containerPort: 80
             - containerPort: 443
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: kiwi-rq-worker
+  labels:
+    app: kiwi
+spec:
```

Comment:
Is this deployment meant to be a singleton?

Reply:
Yeah, as of right now that only time we are using the worker is when we sync the issues when we configure the project.  If we start utilizing the queues more than we probably want to revisit it.